pkg/provider/gitlab.go

```diff
@@ -145,6 +181,10 @@ func (repo *GitLabRepository) GetCommits(fromSha, toSha string) ([]*semrel.RawCo
 }
 
 func (repo *GitLabRepository) GetReleases(rawRe string) ([]*semrel.Release, error) {
+	if repo.useJobToken {
```

Comment:
Is there a specific reason to not use the release api endpoint and parse the tags instead ? 
If not, we should change `GetReleases` to request the releases from the api endpoint instead of listing tags which the job token is permitted to

Reply:
If it's a sparse checkout tags won't be included